gyms.py

```diff
@@ -8,6 +8,7 @@
 import config
 import db
 import utils
+from decimal import *
```

Comment:
Sorry, I'm not used to python. I was justing looking for a quick fix to these 0.0%. What should I do then?